src/guide/send_payment.md

```diff
@@ -75,9 +91,10 @@ try {
 ```python
 bolt11 = "..."
 try:
-  # The `amountSats` param is optional so None can be passed if the 
-  # bolt11 invoice spesifies an amount.
-  sdk_services.send_payment(bolt11=bolt11, amount_sats=None)
+  # The `amount_msat` param is optional and should only passed if the bolt11 doesn't specify an amount.
+  # The amount_msat is required in case an amount is not specified in the bolt11 invoice'.
+  req = SendPaymentRequest(bolt11=bolt11, amount_msat=None)
```

Comment:
Looking at the other docs, this should be `breez_sdk.SendPaymentRequest(bolt11=bolt11, amount_msat=None)`

I think the parameters should also match in all languages, so amount_msat should be 3000000.

Reply:
Addressed 057702b7616d3c5cf7d769031e0044aee16f71ce

Reply:
I'm in favor of @ubbabeck's suggestion with this one.
> Going back on the `amountMsats`, I think it might be best to set all amount_msat as None as in the rust example since bolt11 invoices with an amount is the most used used and safer.

Let's apply the our outcome of this to all examples afterwards.